### PR TITLE
added properties for lostRid api

### PR DESCRIPTION
**Pull request by dhanendra06**

### CODE REVIEW

File: sandbox/registration-processor-mz.properties [hunk 1]
```diff
@@ -128,6 +128,8 @@ RIDGENERATION=http://kernel-ridgenerator-service/v1/ridgenerator/generate/rid
 # We specify Ingress location for dmz which is any worker node and a ingress port
 SYNCSERVICE=${dmz.ingress.base.url}/registrationprocessor/v1/registrationstatus/sync
 PACKETRECEIVER=${dmz.ingress.base.url}/registrationprocessor/v1/packetreceiver/registrationpackets
+# this property is used in lostrid api to get postal code 
+mosip.registration.processor.postalcode.req.url=${dmz.ingress.base.url}/v1/masterdata/registrationcenters
```
Comment (vishwa-vyom) [inline sandbox/registration-processor-mz.properties]: Master data is not running on dmz, please update as below

mosip.registration.processor.postalcode.req.url=${MASTER}/registrationcenters

File: sandbox/registration-processor-mz.properties [hunk 2]
```diff
@@ -128,6 +128,8 @@ RIDGENERATION=http://kernel-ridgenerator-service/v1/ridgenerator/generate/rid
...
```
Comment (vishwa-vyom) [inline sandbox/registration-processor-mz.properties:133]: Also change the below propery in the dmz properties file

mosip.registration.processor.postalcode.req.url=${mz.ingress.url}/v1/masterdata/registrationcenters

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: sandbox/registration-processor-mz.properties (hunk 2 above)
Comment (dhanendra06) [reply sandbox/registration-processor-mz.properties:133]: taken care

File: sandbox/registration-processor-mz.properties (hunk 1 above)
Comment (dhanendra06) [reply sandbox/registration-processor-mz.properties]: taken care

Comment (vishwa-vyom) [reply sandbox/registration-processor-mz.properties]: ok resolved

File: sandbox/registration-processor-mz.properties (hunk 2 above)
Comment (vishwa-vyom) [reply sandbox/registration-processor-mz.properties:133]: I don't see the changes to dmz property file in this PR

Comment (vishwa-vyom) [reply sandbox/registration-processor-mz.properties:133]: ok resolved

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**